cmd/testcase_launch.go

```diff
@@ -219,6 +223,9 @@ func MainTestRunLaunch(client *api.Client, testCaseSpec string, testRunLaunchOpt
 		launchOptions.ClusterSizing = "preflight"
 	}
 
+	// TODO: Validate label input format
```

Comment:
```suggestion
	// TODO: Validate label input format
```

Validations include:

- The key for a label must be unique
- We currently do not allow to attach more than 10 labels to a Test Run

Reply:
The server already validates this; not sure we should do it in the CLI as well.

Reply:
What @zeisss said, lets not validate it for now (can add that later, if we are sure that we won't change the rules).

Reply:
> The server already validates this; not sure we should do it in the CLI as well.

There are validations which currently cannot be performed by the Server, e.g. if the user provides multiple identical labels, in which case "the last ones wins".

```
forge tc launch L6jJp8ll --validate --label foo=bar --label bar=baz --label bar=bax
```

Otherwise, I agree: Relying on the server side validation makes sense, if we are unsure if we want to adjust the Validations in the near future.

Reply:
I'll see if I can adjust the Validation error messages in the forge, so that we generate useful information for the user (see https://github.com/stormforger/forge/pull/3344#issuecomment-865837092)

Reply:
Since we have a `map[string]string` here, we also cannot detect duplicates.

Reply:
The forge error messages got a bit massaged and I removed the TODO (see 20475a9319ee5507e1ae89bce70d206539ff51bb)

=> Resolving this conversation